unitTests/ui/FacetValueSuggestionsTest.ts

```diff
@@ -130,6 +130,78 @@ export function FacetValueSuggestionsTest() {
       });
     });
 
+    describe('when the field is a standard category field', () => {
+      beforeEach(() => {
+        setUpKeywordInOmnibox(aKeyword);
+        setUpOmniboxSuggestionsToReturn([getOmniboxSuggestionValue('a|b|c|d')]);
+        setUpSuggestionsFromProviderToReturn([
+          { field: someField, keyword: aKeyword, numberOfResults: 10, score: { distanceFromTotalForField: 100 }, value: 'a|b|c|d' }
+        ]);
+        test.cmp.options.isCategoryField = true;
+      });
+
+      it('should modify the state by correctly splitting the field value', async done => {
+        const resultingArgs = await triggerPopulateOmniboxEvent();
+
+        firstSuggestion(resultingArgs).then(result => {
+          result[0].onSelect();
+          expect(test.env.queryStateModel.set).toHaveBeenCalledWith(QueryStateModel.attributesEnum.fv, {
+            [someField]: ['a', 'b', 'c', 'd']
+          });
+          done();
+        });
+      });
+    });
+
+    describe('when the field is a category field', () => {
+      beforeEach(() => {
+        setUpKeywordInOmnibox(aKeyword);
+        setUpOmniboxSuggestionsToReturn([getOmniboxSuggestionValue()]);
+        test.cmp.options.isCategoryField = true;
+      });
+
+      describe('with standard delimiter', () => {
+        beforeEach(() => {
+          setUpSuggestionsFromProviderToReturn([
+            { field: someField, keyword: aKeyword, numberOfResults: 10, score: { distanceFromTotalForField: 100 }, value: 'a|b|c|d' }
+          ]);
+        });
+
+        it('should modify the state by correctly splitting the field value', async done => {
```

Comment:
How is this test different from the one on line 143?

Reply:
Should be fixed after removing bad copy + paste :)